packages/c/cgal/pspec_x86_64.xml

```diff
@@ -25,12 +26,11 @@
             <Path fileType="executable">/usr/bin/cgal_create_CMakeLists</Path>
             <Path fileType="executable">/usr/bin/cgal_create_cmake_script</Path>
             <Path fileType="executable">/usr/bin/cgal_make_macosx_app</Path>
-            <Path fileType="doc">/usr/share/doc/CGAL-5.0.2/AUTHORS</Path>
-            <Path fileType="doc">/usr/share/doc/CGAL-5.0.2/CHANGES.md</Path>
-            <Path fileType="doc">/usr/share/doc/CGAL-5.0.2/LICENSE</Path>
-            <Path fileType="doc">/usr/share/doc/CGAL-5.0.2/LICENSE.FREE_USE</Path>
-            <Path fileType="doc">/usr/share/doc/CGAL-5.0.2/LICENSE.GPL</Path>
-            <Path fileType="doc">/usr/share/doc/CGAL-5.0.2/LICENSE.LGPL</Path>
+            <Path fileType="doc">/usr/share/doc/CGAL/AUTHORS</Path>
```

Comment:
Let's delete these  doc's. They are not necessary.
Please add a line in `package.yml` in the end of  `install` section:
`rm -rf $installdir/usr/share/doc`